src/script/builder/tir/prim_func_frame.cc

```diff
@@ -74,6 +74,27 @@ tvm::tir::Buffer Arg(String name, tvm::tir::Buffer buffer) {
 
 TVM_REGISTER_NODE_TYPE(PrimFuncFrameNode);
 
+TVM_REGISTER_GLOBAL("script.builder.tir.PrimFuncFrame")
+  .set_body_typed([](String name){
+    return PrimFunc_(name);
+  });
+
+TVM_REGISTER_GLOBAL("script.builder.tir.EnterPrimFuncFrame")
+  .set_body_method<PrimFuncFrame>(&PrimFuncFrameNode::EnterWithScope);
+
+TVM_REGISTER_GLOBAL("script.builder.tir.ExitPrimFuncFrame")
+  .set_body_method<PrimFuncFrame>(&PrimFuncFrameNode::ExitWithScope);
+
+TVM_REGISTER_GLOBAL("script.builder.tir.ArgVar")
+  .set_body_typed([](String name, tvm::tir::Var var){
+    Arg(name, var);
+  });
+
+TVM_REGISTER_GLOBAL("script.builder.tir.ArgBuffer")
+  .set_body_typed([](String name, tvm::tir::Buffer buffer){
+    Arg(name, buffer);
+  });
```

Comment:
```suggestion
TVM_REGISTER_GLOBAL("script.builder.tir.Arg")
  .set_body_typed([](String name, ObjectRef obj){
    using namespace tvm::tir;
    if (const auto* var = obj.as<VarNode>()) {
      Arg(name, GetRef<Var>(var));
    } else if (const auto* buffer = obj.as<BufferNode>()) {
      Arg(name, GetRef<Buffer>(buffer));
    } else {
      // error out
    }
  });
```

Reply:
Do we need to return `Arg` here? If so, how to return two different types? Maybe cast them into `ObjectRef` together?